flang/test/Lower/OpenMP/Todo/masked-directive.f90

```diff
@@ -0,0 +1,65 @@
+
+! This test checks lowering of OpenMP masked Directive.
+
+// RUN: not flang-new -fc1 -emit-fir -fopenmp %s 2>&1 | FileCheck %s
```

Comment:
Please use `%not_todo_cmd` to ensure this works correctly in all kind of builds.

Reply:
Done. Thank you!